File: 프로그래머스/lv0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    flag = True
    for d in db:
        a_id, a_pw = d
        if id_pw[0] == a_id:
            if id_pw[1] == a_pw:
                return "login"
            else:
                flag = False
    
    if flag:
        return "fail"
    else:
        return "wrong pw"
        
        

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.java
class Solution {
    public String solution(String[] id_pw, String[][] db) {
        String id = id_pw[0];
        String password = id_pw[1];
        
        for (String[] user : db) {
            if (user[0].equals(id) && user[1].equals(password)) {
                return "login";
            } else if (user[0].equals(id)) {
                return "wrong pw";
            }
        }
        return "fail";
    }
}